spec/acceptance/target_spec.rb

```diff
@@ -41,4 +42,29 @@
       end
     end
   end
+
+  route 'api/v1/targets/:id', 'Delete target' do
+    delete 'Delete' do
+      context 'when target belongs to user' do
+        let(:request) { { id: target.id } }
+
+        example 'Delete' do
+          do_request(request)
+
+          expect(status).to eq 200
+        end
+      end
+
+      context 'when target does not belong to user' do
+        let(:new_target) { create(:target) }
+        let(:request) { { id: new_target.id } }
+
+        example 'Delete' do
```

Comment:
Should we rename this `example` to clarify that actually the target is not being deleted?